Sample-01/.env.local.example

```
@@ -1,7 +1,5 @@
 AUTH0_SECRET=replace-with-your-own-secret-generated-with-openssl
-AUTH0_BASE_URL=http://localhost:3000
-AUTH0_ISSUER_BASE_URL='https://{DOMAIN}'
-AUTH0_CLIENT_ID='{CLIENT_ID}'
-AUTH0_CLIENT_SECRET='{CLIENT_SECRET}'
-AUTH0_AUDIENCE=
-AUTH0_SCOPE='openid profile'
+AUTH0_CLIENT_ID={CLIENT_ID}
+AUTH0_CLIENT_SECRET={CLIENT_SECRET}
+AUTH0_DOMAIN={DOMAIN}
+APP_BASE_URL=http://localhost:3000
```

```suggestion
APP_BASE_URL=http://localhost:3000
AUTH0_DOMAIN={DOMAIN}
AUTH0_CLIENT_ID={CLIENT_ID}
AUTH0_CLIENT_SECRET={CLIENT_SECRET}
```

Doing it this way simplifies the diff.